src/main/minio.js

```diff
@@ -1590,10 +1590,10 @@ export class Client {
   // * `respHeaders` _object_: response headers to override (optional)
   presignedGetObject(bucketName, objectName, expires, respHeaders, cb) {
     if (!isValidBucketName(bucketName)) {
-      throw new errors.InvalidBucketNameError('Invalid bucket name: ' + bucketName)
+      throw new errors.InvalidBucketNameError('Invalid bucket name: ${bucketName}')
```

Comment:
It should be within a backtick instead of single-quotes.

`throw new errors.InvalidBucketNameError('Invalid bucket name: ${bucketName}')`